examples/federated-learning/tff/nvflare/docker-image/Dockerfile

```diff
@@ -0,0 +1,8 @@
+#checkov:skip=CKV_DOCKER_3: No need for a user
+#checkov:skip=CKV_DOCKER_2: No need for HEALTHCHECK
+FROM nvflare/nvflare:2.4.1
```

Comment:
You need to add this container image build to the matrix in `.github/workflows/build-container-images.yaml`

Reply:
Done